### ISSUE TITLE
typing "@" opens goto-window instead of typing the character in the editor

### ISSUE THREAD

**Issue by zteli** [Zettlr/Zettlr#4775]
### Description

When I want to insert a citation via the @-character, the character does not appear in the editor window. Instead the "go-to-line"-window opens. So i can't type the @-character in the editor...

Workaround: I copy the @-character from a window in VS-Code, which is a bit cumbersome...

### Reproducing

try to type the @-character in a new or existing file...

### Zettlr Version

Stable (most recent version)

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Ventura 13.6.1

**Comment by zteli** [comment]
addition: I've tried with the MacBookPro-keyboard and with an external keyboard (Logitech), both with the same result...

**Comment by nathanlesage** [comment]
I cannot reproduce this issue; typing an `@`-character inserts that character and, optionally, suggests citations for autocompletion. Can you please confirm that you are on v3.0.2?

**Comment by zteli** [comment]
yes, I am on v3.0.2.

**Comment by nathanlesage** [comment]
Okay. Could you make a screenshot of whatever you are seeing, so that I can understand what your problem could be?

**Comment by zteli** [comment]
I did a screen recording, maybe this is helpful...

https://github.com/Zettlr/Zettlr/assets/99650750/30fe9d64-cb12-48c6-ad95-3fd81d30baf2

**Comment by nathanlesage** [comment]
Thank you very much for the screen recording, that gave me all the info I need – especially the idea to add an on-screen keyboard was extremely helpful.

So, I can see that you'r using a Swiss-German keyboard layout. I just looked it up on the Apple website, and it appears that on Swiss-German keyboards, the @-key is located on the G-key. On German-German keyboards, the @-key is located on the L key. I did some further digging, and saw that CodeMirror (that's the editor framework Zettlr uses) registers the GoTo-Line command using the keyboard shortcut `Alt+G`.

So the problem you are encountering is the following: In order for CodeMirror to listen to keyboard shortcuts, it has to evaluate the pressed keys, and for it, it registers as "Alt-G", which is to be expected. However, since this is *also* used to insert an @-character, this means that you'll never be able to insert an @-character. This is not so much a bug as a technical incompatibility.

This should be comparatively easy to fix, but it may take some time, because I'll have to get in touch with the developers of CodeMirror.
